website/docs/configuration/about.md

```diff
@@ -279,10 +279,25 @@ benthos -c ./your-config.yaml echo
 
 You can check the output of the above command to see if certain sections are missing or fields are incorrect, which allows you to pinpoint typos in the config.
 
+## Shutting down
+
+Under normal operating conditions, the Benthos process will shut down when there are no more messages produced by inputs and the final message has been processed. The shutdown procedure can also be initiated by sending the process a interrupt (`SIGINT`) or termination (`SIGTERM`) signal. There are two top-level configuration options that control the shutdown behaviour: `shutdown_timeout` and `shutdown_delay`.
+### Shutdown delay
+
+The `shutdown_delay` option can be used to delay the start of the shutdown procedure. This is useful for pipelines that will process a finite number of messages and terminate but need a short grace period to have their metrics and traces scraped (or pushed). While the shutdown delay is in effect, the HTTP metrics endpoint continues to be available for scraping and any active tracers are free to flush remaining traces.
```

Comment:
> to have their metrics and traces scraped (or pushed)

Doesn't Benthos attempt to push metrics just before exiting? https://www.benthos.dev/docs/components/metrics/prometheus#push-gateway

Reply:
Ah nice catch! I remove the `(or pushed)` wording but you've also touched on a subject that I should at least discuss in the PR description.